mifosng-android/src/main/java/com/mifos/mifosxdroid/online/note/NoteFragment.kt

```diff
@@ -21,115 +24,33 @@ import javax.inject.Inject
  * Created by rahul on 4/3/17.
  */
 @AndroidEntryPoint
-class NoteFragment : MifosBaseFragment(), OnRefreshListener {
+class NoteFragment : MifosBaseFragment() {
 
-    private lateinit var binding: FragmentNotesBinding
     private val arg: NoteFragmentArgs by navArgs()
 
-    private lateinit var viewModel: NoteViewModel
-
-    @JvmField
-    @Inject
-    var noteAdapter: NoteAdapter? = null
     private var entityType: String? = null
     private var entityId = 0
-    private var notes: List<Note> = ArrayList()
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         entityId = arg.entiyId
         entityType = arg.entityType
     }
 
-    override fun onResume() {
-        super.onResume()
-        viewModel.loadNote(entityType, entityId)
-    }
-
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
-        binding = FragmentNotesBinding.inflate(inflater, container, false)
-        viewModel = ViewModelProvider(this)[NoteViewModel::class.java]
-        showUserInterface()
-        viewModel.loadNote(entityType, entityId)
-        viewModel.noteUiState.observe(viewLifecycleOwner) {
-            when (it) {
-                is NoteUiState.ShowEmptyNotes -> {
-                    showProgressbar(false)
-                    showEmptyNotes()
-                }
-
-                is NoteUiState.ShowError -> {
-                    showProgressbar(false)
-                    showEmptyNotes()
-                }
-
-                is NoteUiState.ShowNote -> {
-                    showProgressbar(false)
-                    showNote(it.note)
-                }
-
-                is NoteUiState.ShowProgressbar -> showProgressbar(true)
-                is NoteUiState.ShowResetVisibility -> {
-                    showProgressbar(false)
-                    showResetVisibility()
-                }
+        toolbar?.visibility = View.GONE
+        return ComposeView(requireContext()).apply {
+            setContent {
```

Comment:
setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
add this line above setContent